src/map.rs

```diff
@@ -97,13 +149,14 @@ impl Map {
             image_layers,
             object_groups,
             properties,
-            background_colour: c,
+            background_color: c,
             infinite: infinite.unwrap_or(false),
+            source: map_path.and_then(|p| Some(p.to_owned())),
         })
     }
 
     /// This function will return the correct Tileset given a GID.
-    pub fn get_tileset_by_gid(&self, gid: u32) -> Option<&Tileset> {
+    pub fn tileset_by_gid(&self, gid: u32) -> Option<&Tileset> {
```

Comment:
This rename should be reflected in the `README.md` as well.

Reply:
I didn't change the README at all, all changes are in the changelog.